packages/renderer/src/lib/node/NodeDetails.svelte

```diff
@@ -1,17 +1,19 @@
 <script lang="ts">
-import type { V1Node } from '@kubernetes/client-node';
+import type { CoreV1Event, KubernetesObject, V1Node } from '@kubernetes/client-node';
 import { StatusIcon, Tab } from '@podman-desktop/ui-svelte';
-import { onMount } from 'svelte';
+import { onDestroy, onMount } from 'svelte';
 import { router } from 'tinro';
 import { stringify } from 'yaml';
 
 import { kubernetesCurrentContextEvents, kubernetesCurrentContextNodes } from '/@/stores/kubernetes-contexts-state';
 
+import type { IDisposable } from '../../../../main/src/plugin/types/disposable';
```

Comment:
Isn't it an issue to get the Disposable type from the server side there ?

Reply:
Right. But I wonder which one to use in `listenReource`. The native `Disposable`? The `Disposable` provided by podman-desktop/api?

Reply:
I mean, it should be in a package directly available in case we would like to use such pattern on the client side

for now this type is only a server side type

Reply:
I've created the issue https://github.com/podman-desktop/podman-desktop/issues/11311 to work on this